sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -530,7 +535,7 @@ public void run() {
         }
 
         /* Nothing to send. Some corrupted entries may be deleted, reset the counter. */
-        groupState.mPendingLogCount = 0;
+        groupState.mPendingLogCount = mPersistence.countLogs(groupState.mName);
```

Comment:
Can you explain?
Also the comment should be updated.

Reply:
Its not that we get fitting logs for existing authTokens, its the opposite - we iterate through tokens, and send logs that fit time bounds. Since we are adding a callback on token refresh (when we find expired token in iteration), we need to make sure we sent all the logs available - so we need to get that count and put it to mPendingLogCount.

Reply:
@Jamminroot Thanks